server/area_manager.py

```diff
@@ -104,6 +104,8 @@ def new_client(self, client):
         def remove_client(self, client):
             """Remove a disconnected client from the area."""
             self.clients.remove(client)
+            if client in self.afkers:
+                self.afkers.remove(client)
```

Comment:
So why doesn't [this same exact code](https://github.com/AttorneyOnline/tsuserver3/blob/7b5adc8fa2ca0f06afcad860060938713d56babb/server/client_manager.py#L591) work?